app/src/main/res/values/styles.xml

```diff
@@ -77,6 +77,19 @@
         <item name="android:paddingEnd">10dp</item>
     </style>
 
+    <style name="TVOnboardingButton">
+        <item name="android:background">@drawable/confirm_button_background</item>
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">37dp</item>
+        <item name="android:layout_gravity">center</item>
+        <item name="android:textSize">14sp</item>
+        <item name="android:textAllCaps">false</item>
+        <item name="android:textColor">#FFF9F9FA</item>
+        <item name="android:fontFamily">@string/font_ember_regular</item>
+        <item name="android:paddingStart">10dp</item>
+        <item name="android:paddingEnd">10dp</item>
+    </style>
```

Comment:
Optional nit: since this is only used in one place, I would probably inline it instead of making a style.